io/plugins/eu.esdihumboldt.hale.io.xls/plugin.xml

```diff
@@ -144,6 +155,15 @@
             </parameterBinding>
          </providerParameter>
       </provider>
+      <provider
```

Comment:
I guess this was added by accident?

Reply:
Yes. It was just a test. I'll remove it.